Bio/Phylo/NewickIO.py

```diff
@@ -287,18 +288,60 @@ def to_strings(
         )
 
         def newickize(clade):
-            """Convert a node tree to a Newick tree string, recursively."""
-            label = clade.name or ""
-            if label:
-                unquoted_label = re.match(token_dict["unquoted node label"], label)
-                if (not unquoted_label) or (unquoted_label.end() < len(label)):
-                    label = "'%s'" % label.replace("\\", "\\\\").replace("'", "\\'")
-
-            if clade.is_terminal():  # terminal
-                return label + make_info_string(clade, terminal=True)
-            else:
-                subtrees = (newickize(sub) for sub in clade)
-                return "(%s)%s" % (",".join(subtrees), label + make_info_string(clade))
+            """Convert a node tree to a Newick tree string, iteratively.  This is done
```

Comment:
Put a blank line after "..., iteratively." to follow the PEP257 docstring convention. This is important for extracting the docstrings for API documentation.

Reply:
Fixed!